plat/hisilicon/hikey/aarch64/hikey_common.c

```diff
@@ -28,7 +28,7 @@
 					TSP_SEC_MEM_SIZE,		\
 					MT_MEMORY | MT_RW | MT_SECURE)
 
-#if LOAD_IMAGE_V2
+#if LOAD_IMAGE_V2 && (HIKEY_TSP_RAM_LOCATION_ID == HIKEY_SRAM_ID)
```

Comment:
`HIKEY_OPTEE_PAGEABLE_LOAD_*` are not defined for SRAM use atm, so maybe just remove `MAP_OPTEE_PAGEABLE` from `hikey_mmap` until they're defined for SRAM use?

Same for below.

Reply:
OK. I'll remove it.